.gitignore

```diff
@@ -41,5 +41,9 @@ node_modules
 
 package-lock.json
 
+# Browser Version
 browser/lib/*.js
```

Comment:
I think the files should be deleted if they are in .gitignore (.gitignore only applies to additions, not to modifications)

Reply:
The file browser/lib/jimp.min.js is still in the repository in your Pull Request : 

https://github.com/hipstersmoothie/jimp/blob/00093b99f32c1f71cc9be4aea613a0e0e52e7a45/browser/lib/jimp.min.js